temp.graphql

```diff
@@ -0,0 +1,1046 @@
+query Mainnet {
+  #162633775975 - 162633775975 = 0
+  tok0sent: subgraph_send_aggregate(where: {token: {_eq: "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599"}, _gs_chain: {_eq: "mainnet"}}, distinct_on: id) {
```

Comment:
it should be possible to `groupBy: token` (in the same section where you're currently doing "aggregate") so you can have a single query for `send` and a single query for `receive`. then you can probably even take it a step further and filter something like `where: token: {_in: ["0xtoken1..., 0xtoken2..."]}`

Reply:
if there are only two queries, it's easier to review each of them very closely. but since each of the tokens' queries are identical, it's also easy to put on blinders and just review one token's query as the general case :)

Reply:
Got it - combining all the queries asap! Will have this in the morning.

Reply:
Hm, so groupBy is somehow not a thing: 
<img width="427" alt="Screen Shot 2022-11-08 at 8 23 20 AM" src="https://user-images.githubusercontent.com/76530366/200576008-23486ddd-ba33-4ac6-957b-1e253640b1c1.png">
Will indeed attempt the filter method though; I think I can recombine a lot of them into one.

Reply:
Apparently, groupBy is not possible.

As to how the aggregates work, I'm not sure if there is another way around besides for splitting them into individual aggregate queries like I did previously...

Though, summing the token send / receives into larger aggregates, total received is clearly greater than total sent (if that provides some insight).

Will add another query for a non-affected token as a sanity check.